providers/agent/mcorpc/audit/choria_test.go

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) 2020-2021, R.I. Pienaar and the Choria Project contributors
 //
 // SPDX-License-Identifier: Apache-2.0
+//go:build !windows
+// +build !windows
```

Comment:
I couldn't find another way to avoid Windows tests failing because using `syscall.Stat_t` to validate with unit tests, since seems the code still has to compile even when wrapped in `runtime.GOOS != "windows"`.

Reply:
make a file `choria_test_windows.go` inside it 

```
func checkFileGid(nfo *os.FileInfo) { }
```

and in `choria_test_unix.go`

```
func checkFileGid(nfo *os.FileInfo) {
  // do the check 
}
```

and from this file just call `checkFileGid(nfo)` it will then compile fine everywhere just noop on windows

Reply:
Implemented this suggestion